example/viam_example_app/lib/main.dart

```diff
@@ -92,8 +94,8 @@ class _MyHomePageState extends State<MyHomePage> {
       _loading = true;
     });
     final robotFut = RobotClient.atAddress(
-      '<URL>',
-      RobotClientOptions.withLocationSecret('<SECRET>'),
+      dotenv.env['ROBOT_LOCATION'] ?? '',
```

Comment:
what would be stored in these env files? would a user upload their robot_location to the file or is this just a more professional way to include input that may change depending on user?

Reply:
Usually, environment-specific variables are stored in env files (in this case, the robot location and secret). This is super useful when you have different variables based on launch environment (e.g. debug vs release). This is useful in this example because it's also how we're testing internally, so it let's us set these values without the fear of committing them.